pkg/command/context.go

```diff
@@ -632,6 +636,8 @@ func doCSPAuthAndUpdateContext(c *configtypes.Context, endpointType string) (cla
 	if apiTokenExists {
 		log.Info("API token env var is set")
 	} else {
+		fmt.Fprintln(os.Stderr)
+		log.Info("The API key can be provided by setting the TANZU_API_TOKEN environment variable")
```

Comment:
could this go directly into `promptAPIToken()`?

Reply:
I think better to keep it out, we don't all `promptAPIToken()` users want to log the message about API key, better keep here.